test/doc/core/bit_ternary.cpp

```diff
@@ -0,0 +1,35 @@
+// revision 0
+#include <eve/module/core.hpp>
+#include <iostream>
+
+template < int a, int b, int c, int d, int e, int f , int g, int h>
```

Comment:
Yes, this should be public API and need a proper name and API

Reply:
done eve;;truth_table